src/+otp/+sanzserna/fforcing.m

```diff
@@ -0,0 +1,10 @@
+function dy = fforcing(t, y, ~, x)
+
+hIn = length(y);
```

Comment:
Minor point, but length could return the wrong thing here if the input is vectorized. size(y, 1) would probably be more correct here.

Reply:
True.  This probably appears in many other problems as well.  It shouldn't matter unless vectorization is set to 'on' in the Rhs, but is still worth fixing.